helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml

```diff
@@ -128,7 +128,8 @@ spec:
                                 description: Enable or disable TLS hostname verification. Default value is `false`.
                               enableECDSA:
                                 type: boolean
-                                description: Enable or disable ECDSA support by installing BouncyCastle crypto provider. Default value is `false`.
+                                description: Enable or disable ECDSA support by installing BouncyCastle crypto provider. ECDSA support is always enabled.
```

Comment:
This file should not be changed - this is the CRD from 0.23 (all changes should be done to `packaging/...`). Can you please revert it?

Reply:
Hard to keep check on these rules ... I reverted.